src/compat/libc/stdlib/mbstowcs.c

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file
+ * @brief mbstowcs function.
+ *
+ * @see stdlib.h
+ *
+ * @date 01.11.18
+ * @author Chubukov Filipp
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <defines/size_t.h>
+#include <defines/wchar_t.h>
+
+size_t mbstowcs(wchar_t *wcstring, const char *mbstring, size_t max) {
```

Comment:
If `mbstring` contains terminating null character, it should be converted too, but that's not happening.

By the way, tests failed to check it, so they should be fixed as well.